sql/paypal_transactions.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE `paypal_transactions` (
```

Comment:
I'm not sure if "transactions" is the proper term. Maybe "items" or "order_items" since `custom_id` is for each item?